other/docker/misra/Makefile

```diff
@@ -155,7 +155,12 @@ SUPPRESSIONS += 21.10
 
 CPPFLAGS := -DCMP_NO_FLOAT=1 -DMIN_LOGGER_LEVEL=LOGGER_LEVEL_TRACE
 
-SOURCES := $(shell find /src/workspace/c-toxcore -name "*.c")
+FIND_FLAGS := -name "*.c"								\
+			  -and -not -wholename "*/auto_tests/*"		\
+			  -and -not -wholename "*/other/*"			\
```

Comment:
Doesn't that exclude the bootstrap daemon and other utilities when it didn't before? Do we not want to run misra on them?

Reply:
It does, and [no we don't](https://github.com/TokTok/c-toxcore/blob/d6d67d56f3e56d30cefd81f70e68c041a8c4bf2a/other/docker/misra/Dockerfile#L15-L19).